src/Authentication/Authentication.Core/Microsoft.Graph.Authentication.psm1

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This file should be removed. Authentication.Core is just a core assembly and not a module.

Reply:
I assumed the file was from your re-org PR